### ISSUE TITLE
IntelliJ Rust plugin support (cfg_attr)

### ISSUE THREAD

**Issue by Gohla** [hecrj/iced#104]
Currently, the [IntelliJ Rust plugin](https://intellij-rust.github.io/) does not support `cfg_attr`, meaning that modules such as:

```rust
#[cfg_attr(target_arch = "wasm32", path = "web.rs")]
#[cfg_attr(not(target_arch = "wasm32"), path = "native.rs")]
mod platform;
```

are not handled correctly by the plugin's analysis. This causes unresolved references and code completion to not work, which makes developing against this (really cool!) library a bit annoying. The plugin does accept normal `cfg` attributes as follows:

```rust
#[cfg(target_arch = "wasm32")]
#[path = "web.rs"]
mod platform;
#[cfg(not(target_arch = "wasm32"))]
#[path = "native.rs"]
mod platform;
```

which unfortunately has a bit more boilerplate. Would you consider changing this? If so, I could create a PR with these changes.

**Comment by hecrj** [comment]
Yes, I'd be open to change this while the IntelliJ plugin gets there.

There are [multiple occurrences of `cfg_attr`](https://github.com/hecrj/iced/search?q=cfg_attr&unscoped_q=cfg_attr) in the codebase. Feel free to open a PR fixing them!

**Comment by hecrj** [comment]
Closed by #106.
